CHANGES

```diff
@@ -1,4 +1,6 @@
 [1.9.10]
+- API Addition: get ID for game controllers assigned by backend
```

Comment:
This should go under 1.9.11 and not 1.9.10 I guess.

Reply:
That was a mistake. I moved it to 1.9.11.